tests/ctest_helpers.cmake

```diff
@@ -91,6 +91,20 @@ function(build_pmemobj_cow_check)
 			OUTPUT_QUIET)
 endfunction()
 
+function(build_check_is_pmem)
+	execute_process(COMMAND ${CMAKE_COMMAND}
```

Comment:
Does it really have to be standalone program? We can check the file system status at the time of test execution.